.buildkite/pipeline.yml

```diff
@@ -11,43 +11,55 @@ env:
 
 steps:
   - label: hlint
+    key: hlint
     if: &not_scheduled_autodoc
       build.branch != "autodoc/master" && build.branch != "master" && build.source != "schedule"
     commands:
     - nix run -f ci.nix pkgs.hlint -c
         ./scripts/lint.sh
 
   - label: reuse lint
+    key: reuse_lint
     if: *not_scheduled_autodoc
     commands:
     - nix run -f ci.nix pkgs.reuse -c
         reuse lint
 
   - label: check trailing whitespace
+    key: check_trailing_whitespace
     if: *not_scheduled_autodoc
     commands:
     - .buildkite/check-trailing-whitespace.sh
 
   - label: crossref-verify
+    key: crossref-verify
     if: *not_scheduled_autodoc
     commands:
     - nix run -f ci.nix crossref-verifier -c
         crossref-verify --mode local-only --config ./.crossref-verifier.yaml
-    soft_fail: true  # TODO: remove
+    soft_fail: true
```

Comment:
It think the problem here is suppose to get solved after an update to the verifier and eventually we won't need the `soft_fail` anymore.

So let's leave the TODO (here and the one below).